timescaledb/how-to-guides/compression/backfill-historical-data.md

```diff
@@ -96,12 +96,12 @@ CALL run_job(<job_id>);
 ## Future Work [](future-work)
 
 One of the current limitations of TimescaleDB is that once chunks are converted
-into compressed column form, we do not currently allow further modifications
-of the data (e.g., inserts, updates, deletes) or the schema without manual decompression,
-except as noted [above](#compression-schema-changes). In other words, chunks are
-immutable in compressed form. Attempts to modify the chunks' data will either
-error or fail silently (as preferred by users). We plan to remove this limitation
-in future releases.
+into compressed column form, we do not currently allow updates and deletes
+of the data or changes to the schema without manual
+decompression, except as noted [above](#compression-schema-changes). In other
```

Comment:
compression-schema-changes link appears to be broken in the docs and needs fixing up as well.

Reply:
@Loquacity do you know how we can fix this?

Reply:
It looks like you're writing a link to an anchor tag, but I think you want to point to another page. 

```suggestion
decompression, except as noted [above][compression-schema-changes]. In other
```